Artyssee/cantine-choupi
# Hotfix testing sass

## Changes

(describe what has been changed in this PR. [+] for each addition, [-] for removals and [*] for changes)

[ * ] Rebuild the project with the correct version of jest and the correct test commands so testing is possible again.
[ * ] Added the possibility of sass imports and the correct structure for sass modules

## Testing steps

How to test:

1. import the SASS in a page file and start using sass modules
2. Check if the styling is applied correctly
3. run `npm run test` from the terminal and see if jest works. Tests don't have to be correct as this is a fix for testing in general.
